src/roles/ha_db_hana/tasks/block-network.yml

```diff
@@ -56,7 +56,7 @@
           delegate_to:                      localhost
           ansible.builtin.shell: |
                                             for i in $(seq 1 30); do
-                                              if ! ping -c 1 -w 1 {{ ansible_host }}; then
+                                              if ! nc -zv -w1 {{ ansible_host }} {{ sap_port_to_ping }}; then
```

Comment:
Do we need to ensure that nc is available/installed?

Reply:
Based on the [documentation ](https://linuxize.com/post/netcat-nc-command-with-examples/) about nc (netcat), yes it is pre-installed on Linux distributions like Ubuntu.